src/main/java/ru/sbt/mipt/oop/smart/actions/ActionLightOn.java

```diff
@@ -0,0 +1,15 @@
+package ru.sbt.mipt.oop.smart.actions;
+
+import ru.sbt.mipt.oop.smart.devices.Light;
+import ru.sbt.mipt.oop.smart.devices.SmartDevice;
+import ru.sbt.mipt.oop.smart.devices.SmartDeviceType;
+
+public class ActionLightOn implements Action {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D1%80%D0%B8%D0%B4%D0%B5%D1%80%D0%B6%D0%B8%D0%B2%D0%B0%D0%B9%D1%82%D0%B5%D1%81%D1%8C-%D0%BF%D1%80%D0%B5%D0%B4%D0%BB%D0%BE%D0%B6%D0%B5%D0%BD%D0%BD%D1%8B%D0%BC-%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B0%D0%BC-action-%D0%B8-actionable